test/e2e/specs-jetpack-calypso/wp-jetpack-connect-spec.js

```diff
@@ -194,8 +192,8 @@ describe( `Jetpack Connect: (${ screenSize })`, function() {
 			return await pickAPlanPage.selectFreePlanJetpack();
 		} );
 
-		step( 'Is redirected back to the Jetpack dashboard with Jumpstart displayed', async function() {
-			return await this.wpAdminJetpack.jumpstartDisplayed();
+		step( 'Can then see the Jetpack plan page in Calypso', async function() {
+			return await PlansPage.Expect( driver );
```

Comment:
This flow would then continue with onboarding checklist and returning to WP admin, but I'd rather PR those tests separately since it's a lot more work.